test/system/800-config.bats

```diff
@@ -253,7 +253,7 @@ EOF
     skip_if_remote "containers.conf does not effect client side of --remote"
 
     CONTAINERS_CONF=/dev/null run_podman push --help
-    assert "$output" =~ "--compression-format string.*compression format to use \(default \"gzip\"\)" "containers.conf should set default to gzip"
+    assert "$output" =~ "--compression-format string.*compression format to use \(default \"zstd\"\)" "containers.conf should set default to zstd"
```

Comment:
Can we change this to a global environment variable?

Reply:
Change what? The test to use a variable? (What would that change?) Introduce a new environment variable interpreted by Podman proper?

Reply:
Something like the following:
assert "$output" =~ "--compression-format string.*compression format to use \(default \"${COMPRESSION_FORMAT}\"\)" "containers.conf should set default to ${COMPRESSION_FORMAT}"

and helpers.bash sets COMPRESSION_FORMAT=gzip|zstd|zstd:chunked

Reply:
Are users really going to set this in the environment? My guess is that most should never want to touch the option if we do our job perfectly.

Here, I’d rather have a more complex test (to the point of grepping the source code for the default) than one more unexpected environment variable (with a name that doesn’t suggest any Podman relationship at all) affecting Podman’s operation, and documented, and maintained forever.

Reply:
I just want this for the test, so that we can move forward with the test and easily swap out compression types.

Reply:
Or is the compression format set within the containers.conf?

Reply:
Ok I get it now, this one should really be merged with the previous DO NOT MERGE hack.

Reply:
I'm not participating in this thread, too much going on, but one request:
```bash
    assert "this" = "expected" "blah blah blah = expected"
```
...you do not need to repeat "expected" in the description message. The description is *only* shown on test failure. And if there's a failure, you are guaranteed to see both actual and expected values. Save yourself the redundancy and maintenance burden.

Reply:
@edsantiago thanks, simplified.